### PR TITLE
bug: fix annotation bug

### PR DESCRIPTION
**Pull request by cdrage**
bug: fix annotation bug

#### What this PR does / why we need it:

When supplying a label in the compose file, it should stay in the output
too.

#### Which issue(s) this PR fixes:

Fixes https://github.com/kubernetes/kompose/issues/1885

#### Special notes for your reviewer:

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

**Comment by cdrage** [comment]
I also fixed the test here: https://github.com/kubernetes/kompose/blob/main/script/test/fixtures/configmap-file-configs/compose-3.yaml

We cannot do this on mac M1 testing since /var/lib/docker isn't available. So I changed it to use /tmp in this PR

File: script/test/cmd/tests.sh [hunk 1]
```diff
@@ -386,3 +386,8 @@ os_cmd="kompose -f $KOMPOSE_ROOT/script/test/fixtures/envvars-with-status/compos
 os_output="$KOMPOSE_ROOT/script/test/fixtures/envvars-with-status/output-os.yaml"
 convert::expect_success "$k8s_cmd" "$k8s_output" || exit 1
 convert::expect_success "$os_cmd" "$os_output" || exit 1
+
+# Test label in compose.yaml appears in the output annotation
+k8s_cmd="kompose -f $KOMPOSE_ROOT/script/test/fixtures/label/compose.yaml convert --stdout --with-kompose-annotation=false"
+k8s_output="$KOMPOSE_ROOT/script/test/fixtures/label/output-k8s.yaml"
+convert::expect_success "$k8s_cmd" "$k8s_output" || exit 1
```
Comment (TessaIO) [inline script/test/cmd/tests.sh:393]: Missing EOF here

File: script/test/fixtures/configmap-file-configs/compose-3.yaml [hunk 2]
```diff
@@ -5,6 +5,4 @@ services:
       - "8081:8080"
       - "8026:8025"
     volumes:
-      - /var/run/docker.sock:/var/run/docker.sock:ro
-      - /sys:/sys:ro
-      - /var/lib/docker/:/var/lib/docker:ro
+      - /tmp:/tmp:ro
```
Comment (TessaIO) [inline script/test/fixtures/configmap-file-configs/compose-3.yaml:8]: Missing EOF

**Review by TessaIO — APPROVED** [review]
LGTM! Just some small nits

**Pull request merged**